spec/models/course/level_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.